gammapy/scripts/image_pipe.py

```diff
@@ -285,7 +285,7 @@ def make_exposure(self, obs_id, spectral_index=2.3, for_integral_flux=False):
         xpix_coord_grid, ypix_coord_grid = exposure.coordinates(coord_type='pix')
         # calculate pixel offset from center (in world coordinates)
         coord = pixel_to_skycoord(xpix_coord_grid, ypix_coord_grid, exposure.wcs, origin=0)
-        center = SkyCoord.from_pixel(self.header["NAXIS1"] / 2., self.header["NAXIS2"] / 2., exposure.wcs)
```

Comment:
Note, that the definition of `SkyMap.center()` might be slightly different (see [here](https://github.com/gammapy/gammapy/blob/master/gammapy/image/maps.py#L271)) from the one used here. Is this covered by a test somehow? If not, could you add at least one assert to the test below? E.g. check that the center coordinates are right...because it might affect science results.